src/components/filter-radio.jsx

```diff
@@ -0,0 +1,71 @@
+import { connect } from 'react-redux';
+
+function handleChange(e, dispatch, action) {
+  return dispatch(action());
+}
+
+function applyActiveClass(children, filter) {
+  return children == filter ? 'active' : ''
+}
+
+let FilterRadio = ({children, action, filter, dispatch}) => (
+    <label className="filter">
+      <input type="radio"
```

Comment:
I don't think we're using the radio buttons here properly. Generally, radio buttons should be grouped and within a form tag. Why do you prefer radio buttons than links?

Reply:
IMO they seem to fit better for the case cause filters are mutually exclusive like the radio buttons.